workflow/snakemake_rules/nextclade.smk

```diff
@@ -88,9 +97,25 @@ rule get_sequences_without_nextclade_annotations:
         fi
         """
 
+
+rule print_number_of_sequences_without_nextclade_annotations:
+    """Print number of sequences in FASTA which don't have clades assigned yet"""
+    input:
+        fasta=f"data/{database}/nextclade.sequences.fasta",
+    output:
+        touch(f"data/{database}/nextclade.sequences.fasta.count"),
+    shell:
+        """
+        echo "[ INFO] Number of sequences to run Nextclade on: $(grep -c '^>' {input.fasta})"
```

Comment:
Adding this so it's easier to see how many sequences nextclade gets to align, helps with debugging etc.